examples/examples/external_struct.rs

```diff
@@ -0,0 +1,72 @@
+use std::sync::Arc;
+
+use rune::runtime::Vm;
+use rune::termcolor::{ColorChoice, StandardStream};
+use rune::{Any, ContextError, Diagnostics, Module};
+
+#[derive(Default, Debug, Any, PartialEq, Eq)]
+#[rune(constructor)]
+struct External {
+    #[rune(get, set)]
+    suite_name: String,
+    #[rune(get, set)]
+    room_number: usize,
+    #[rune(get, set)]
+    is_reserved: bool,
+    #[rune(get, set)]
+    num_beds: usize,
+    #[rune(get, set)]
+    num_baths: usize,
+}
+
+fn main() -> rune::Result<()> {
+    let m = module()?;
+
+    let mut context = rune_modules::default_context()?;
+    context.install(m)?;
+    let runtime = Arc::new(context.runtime());
+
+    let mut sources = rune::sources! {
+        entry => {
+            pub fn main() {
```

Comment:
Add some padding in this test, like a few variable declarations before you construct the struct and it should produce the error which requires adding a stack offset (see above).